zmon_cli/client.py

```diff
@@ -63,6 +63,8 @@ def wrapper(*args, **kwargs):
 
 
 def compare_entities(e1, e2):
+    e1.pop('last_modified', None)
```

Comment:
I thought this was required, since it is a ZMON specific field and entities *shouldn't* include it.